### ISSUE TITLE
Terminal output formatting

### ISSUE THREAD

**Issue by Kubitomakita** [TypistTech/imposter-plugin#86]
I'm not sure if this is a Composer issue, Imposter-plugin issue or PHPCS installer issue, but...

I'm using your awesome `typisttech/imposter-plugin` package as well as the coding standards packages:
- `dealerdirect/phpcodesniffer-composer-installer`
- `phpcompatibility/php-compatibility`
- `wp-coding-standards/wpcs`

The `phpcodesniffer-composer-installer` upon install or update is giving details about installed standards which interferes with imposter output.

This results in the following log:

```
Generating autoload files
Running Imposter...
======================
Loading package information from /var/project-path/composer.json
Imposter operations: 6 transformations
 - Transforming: /var/project-path/vendor/micropackage/requirements/src
 - Transforming: /var/project-path/vendor/micropackage/internationalization/src
 - Transforming: /var/project-path/vendor/micropackage/dochooks/src
 - Transforming: /var/project-path/vendor/micropackage/filesystem/src
 - Transforming: /var/project-path/vendor/micropackage/templates/src
 - Transforming: /var/project-path/vendor/micropackage/templates/src/functions/template.php

Done.
 6/6 [============================] 100%PHP CodeSniffer Config installed_paths set to ../../phpcompatibility/php-compatibility,../../wp-coding-standards/wpcs
Generating optimized autoload filesRunning Imposter...
======================
Loading package information from /var/project-path/composer.json
Imposter operations: 6 transformations
 - Transforming: /var/project-path/vendor/micropackage/requirements/src
 - Transforming: /var/project-path/vendor/micropackage/internationalization/src
 - Transforming: /var/project-path/vendor/micropackage/dochooks/src
 - Transforming: /var/project-path/vendor/micropackage/filesystem/src
 - Transforming: /var/project-path/vendor/micropackage/templates/src
 - Transforming: /var/project-path/vendor/micropackage/templates/src/functions/template.php

Done.
Generated optimized autoload files containing 975 classes
```

**Comment by TangRufus** [comment]
Can you give an example of your `composer.json` (and `phpcs.xml` / `ruleset.xml` if it matters) please?
Goal is to reproduce it on my machine.

**Comment by Kubitomakita** [comment]
Sure thing, you can simply clone my repo on this branch: https://github.com/BracketSpace/Notification/tree/version/7

**Comment by TangRufus** [comment]
To confirm, everything works (namespaces are prefixed and coding standards are insatlled), only issue is the console outputs mixed in 1 line.

Am I correct?

**Comment by Kubitomakita** [comment]
Correct, Imposter output mixed with PHPCS installer output and Imposter gets printed twice.

**Comment by TangRufus** [comment]
Imposter output mixed with PHPCS installer output 

Fixed in #89

> Imposter gets printed twice.

Fixed in #91 

However, seems I ran imposter twice on intentionally.... see: #96

**Comment by TangRufus** [comment]
I was lying.

> Imposter output mixed with PHPCS installer output fixed in #89 

True, fixed in v0.5.0

> Imposter gets printed twice.

Unfortunately, I couldn't find a way to make it work.
For now, I tolerate imposter being ran twice during `compsoer install` and `composer update`.

---

I prefer to publish to packagist.org over wordpress.org nowadays.
Imposter becomes a hobby project for me instead of a mission critical part of the workflow. 
You might want to consider using one of the [alternatives](https://github.com/TypistTech/imposter-plugin#alternatives) instead.

Sorry to give you false hope.

**Comment by Kubitomakita** [comment]
No worries @TangRufus! Thanks for this package. In the current form, it already gives a ton of value.

I'll try to contribute if I'll find some time.
